lib/rspec/matchers/built_in/change.rb

```diff
@@ -72,11 +72,14 @@ def description
           "change #{change_details.value_representation}"
         end
 
-        # @private
         def supports_block_expectations?
           true
         end
 
+        def supports_value_expectations?
```

Comment:
It's possible to tell if a matcher supports block expectations, but it's not possible to detect that it doesn't support value expectations, and will be sending `call` on a value target.

Reply:
Seems reasonable!